docs/creating-manifest/events.md

```diff
@@ -1757,7 +1757,109 @@ onAfterRemoveSSL:
 ```
 @@!
 
+### onBeforeInstallAddon  
+The event is executed before add-on installation  
+
+Event Placeholders:  
+
+-   `${event.params.}`:  
+    - `settings` - ***What is it***  
+    - `envGroups` - environment groups array  
+    - `displayName` - display name for an environment  
+    - `skipNodeEmails` - an ability to skip sending emails about creating nodes. Emails are related only to nodes where implemented reset password functionality  
+    - `source` - ***What is it***  
+    - `nodeGroup` - the defined node layer  
+    - `exportedFromEnv` - ***What is it***  
+    - `envName` - short domain name (without hosting provider URL)  
+    - `actionKey` - ***What is it***  
+    - `id` - add-on id  
+    - `lang` - ***What is it***  
+    - `region` - region, where an environment will be installed  
+    - `targetAppid`  - application identifier  
+-   `${event.response.}`: parameters are absent  
+
+### onAfterInstallAddon  
+The event is executed after add-on installation  
+  
+-   `${event.params.}`:  
+    - `settings` - ***What is it?***  
+    - `envGroups` - environment groups array  
+    - `displayName` - display name for an environment  
+    - `skipNodeEmails` - an ability to skip sending emails about creating nodes. Emails are related only to nodes where implemented reset password functionality  
+    - `source` - ***What is it?***  
+    - `nodeGroup` - the defined node layer  
+    - `exportedFromEnv` - ***What is it?***  
+    - `envName` - short domain name (without hosting provider URL)  
+    - `actionKey` - ***What is it?***  
+    - `id` - add-on id  
+    - `lang` - ***What is it?***  
```

Comment:
user's language code (https://en.wikipedia.org/wiki/Language_code) which was used during the installation. Optional parameter

Reply:
done.